smac/intensification/abstract_racer.py

```diff
@@ -95,14 +101,74 @@ def __init__(self, tae_runner: ExecuteTARun,
         else:
             self.instance_specifics = instance_specifics
 
-    def intensify(self, challengers: typing.List[Configuration],
+        # general attributes
+        self._min_time = 10 ** -5
+        self._num_run = 0
+        self._chall_indx = 0
+        self._ta_time = 0
+
+    def intensify(self, challengers: typing.Optional[typing.List[Configuration]],
```

Comment:
Could you please put both `self` and `challengers` into its own line each? That'll make it easier to spot what's new.